weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -477,30 +480,88 @@ const PanelPlotConfigInner: React.FC<PanelPlotProps> = props => {
             </ConfigPanel.ConfigOption>
           );
         })}
-        {config.series.map((series, i) => {
-          const seriesName = `Series ${i + 1} Name`;
+      </>
+    );
+  }, [config, updateConfig]);
+
+  const newSeriesConfigDom = useMemo(() => {
+    return (
+      <>
+        {config.series.map((s, i) => {
           return (
-            <ConfigPanel.ConfigOption
-              key={seriesName}
-              label={config.series.length > 1 ? seriesName : 'Series'}>
-              <ConfigPanel.TextInputConfigField
-                dataTest={`${seriesName}-label`}
-                value={series.seriesName}
-                label={''}
-                onChange={(event, {value}) => {
-                  updateConfig(
-                    produce(config, draft => {
-                      draft.series[i].seriesName = value;
-                    })
-                  );
-                }}
-              />
-            </ConfigPanel.ConfigOption>
+            <ConfigSection
+              label={`Series ${i + 1}`}
+              menuItems={[
+                {
+                  key: 'Remove series',
+                  content: 'Remove series',
+                  icon: <IconDelete />,
+                  onClick: () => {
+                    updateConfig(PlotState.removeSeries(config, s));
+                  },
+                },
+                {
+                  key: 'Add series from this series',
```

Comment:
In @Cecile0112358's design this was a button displayed more prominently, not something in an overflow menu, did that change? 

https://www.figma.com/file/Vc1PpAp2ymWTJPbsj444D0/Weave-1.1?type=design&node-id=5123-111782&mode=design&t=dwBpZGayGtau1pqA-0

Reply:
do we have a method that creates a new empty Series from scratch? I'm only seeing `PlotState.addSeriesFromSeries `which requires a pre-existing series.

Reply:
Decided on slack to add a button which adds series from latest series

Reply:
added